modules/i18n/src/translations/en-US/portals/user-portal.ts

```diff
@@ -672,7 +672,7 @@ export const userPortal: UserPortalNS = {
             },
             smsOtp: {
                 descriptions: {
-                    hint: "You'll receive a text message containing a one-time verification code"
+                    hint: "You'll receive a text message containing a one-time verification code",
```

Comment:
```suggestion
                    hint: "You'll receive a text message containing an one-time verification code",
```

Reply:
Addressed in https://github.com/wso2/identity-apps/pull/1275/commits/375a1a7a8b040739aaae46cb9b9657e461a0bf70